### ISSUE TITLE
API & SALOON

### ISSUE THREAD

**Issue by nathanlee27** [awtung/p2-cowboys#40]
- [x] Joke API (better understanding of API)
- [x] Format the page better through changing where the games and pictures are located
- [x] Add better spots for the games that fit the theme of the website

Tangibles (video): [API](https://www.loom.com/share/464c94a630234b82bc07dddb7aedff01)
API Code: 
- [Python](https://github.com/awtung/p2-cowboys/blob/424391ff45f1a856562a18c2be6923621e24a869/app/app.py#L117-L138)
- [HTML](https://github.com/awtung/p2-cowboys/blob/c336ee18ef129cfdf83c640b1f14c6b219dea80f/social/home/templates/randomapi.html#L60-L65)

Tangibles (video): [Saloon](https://www.loom.com/share/1aee83a54154422e833dfeae5323bc43)
Saloon Code:
- [HTML](https://github.com/awtung/p2-cowboys/blob/c336ee18ef129cfdf83c640b1f14c6b219dea80f/saloon/templates/saloon.html#L1-L533)

**Comment by Naweid** [comment]
very nice work! showed good understanding of ticket with commits and evidence.

**Comment by andrewc26** [comment]
Good idea! It sounds like you are progressing and improving your knowledge and understanding of topics of coding.

**Comment by taaw** [comment]
I really like the idea about adding games. This ticket has a lot of variety and it would be impressive to see all of it done!

**Comment by MaxVukovich** [comment]
Adding games is a great idea and it is always good to fix the formatting to make it more presentable. The addition of a Rest API is a great idea and will be a major contributed to the project. Ticket looks promising!

**Comment by NoahPidding** [comment]
Seems like some cool ideas for the saloon page. However, maybe be a little more specific in your wording of your goals, and explain how exactly you're going to format the page better and what specific games you are thinking of adding to the page.
